modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordExecManager.kt

```diff
@@ -161,12 +110,12 @@ class MirrordExecManager(private val service: MirrordProjectService) {
         service.versionCheck.checkVersion() // TODO makes an HTTP request, move to background
 
         val cli = cliPath(wslDistribution, product)
-        val config = getConfigPath(mirrordConfigFromEnv)
+        val config = service.configApi.getConfigPath(mirrordConfigFromEnv)
```

Comment:
It looks like it should work, since we no longer need write access, but I'm scared. Have you tried running this on different platforms?

Reply:
As in Goland

Reply:
Tested by Mehul